src/main/java/frc/robot/models/FXGains.java

```diff
@@ -0,0 +1,16 @@
+package frc.robot.models;
+
+public class FXGains {
```

Comment:
These look identical to the SRXGains class.  Let's refactor `SRXGains` to `PhoenixGains` and use that in both `BobTalonSRX` and `BobTalonFX`